hypervisor/common/sched_bvt.c

```diff
@@ -81,10 +161,25 @@ static struct thread_object *sched_bvt_pick_next(__unused struct sched_control *
 
 static void sched_bvt_sleep(__unused struct thread_object *obj)
 {
+	runqueue_remove(obj);
 }
 
 static void sched_bvt_wake(__unused struct thread_object *obj)
 {
+	struct sched_bvt_data *data;
+
+	/* update target not current thread's avt_mcu and evt_mcu */
+	data = (struct sched_bvt_data *)obj->data;
+	/* prevents a thread from claiming an excessive share
+	 * of the CPU after sleeping for a long time as might happen
+	 * if there was no adjustment */
+	data->svt_mcu = get_svt(obj);
+	data->avt_mcu = (data->avt_mcu > data->svt_mcu) ? data->avt_mcu : data->svt_mcu;
```

Comment:
Need we define a threshold for the avt updating in wakeup? This policy should only applied for long sleep object.
Else, every time the woke one might lost a bit of runnable time.